src/main/webapp/index.html

```diff
@@ -47,7 +47,7 @@
 
 <div class="navbar navbar-fixed-top">
     <div class="navbar-inner">
-        <div class="userName-top"><span id="user"></span> | <a href="/logout" id="logout-link">Log out</a></div>
+        <div class="userName-top"><span id="user"></span> | <a href="/v1/logout" id="logout-link">Log out</a></div>
```

Comment:
It will because the link is just a placeholder, the real work is done by JS, overriding the click (see above comment). The JS is doing a `$.ajax` POST` so it will go through the url patching in #11.